gremlin-go/driver/connection_test.go

```diff
@@ -411,4 +414,21 @@ func TestConnection(t *testing.T) {
 			assert.Equal(t, int32(2), val)
 		}
 	})
+
+	t.Run("Test DriverRemoteConnection To Server Configured with Modern Graph", func(t *testing.T) {
```

Comment:
With my incoming change, this should be updated to be a function to skip.

Reply:
Added